renku/core/commands/save.py

```diff
@@ -93,7 +93,13 @@ def repo_sync(repo, message=None, remote=None, paths=None):
     if paths:
         # commit uncommitted changes
         try:
-            repo.git.add(*paths)
+            staged_files = {d.a_path for d in repo.index.diff("HEAD")} if repo.head.is_valid() else set()
```

Comment:
Should we call `git_unicode_unescape` on `d.a_path`? If so, then we should also call it on `paths` as well.

Reply:
We only need to call it on paths that come from git, so yes about adding it here, but `paths` is supplied by us/the user, so there it's not necessary.